sync/etcdv3/etcd.go

```diff
@@ -255,7 +256,7 @@ func (s *Sync) Watch(path string, responseChan chan *sync.Event, stopChan chan b
 	go func() {
 		defer wg.Done()
 		err := func() error {
-			rch := s.etcdClient.Watch(ctx, path, etcd.WithRev(node.Header.Revision+1))
+			rch := s.etcdClient.Watch(ctx, path, etcd.WithRev(revision))
```

Comment:
mostly not your code, but could we use meaningful names instead of wg, rch and wresp?